benchmark/fs/bench-copyFileSync.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bench = common.createBenchmark(main, {
+  type: ['invalid', 'valid'],
+  n: [1e4],
+});
+
+function main({ n, type }) {
+  tmpdir.refresh();
+  const dest = tmpdir.resolve(`copy-file-bench-${process.pid}`);
```

Comment:
I think here you can measure in two ways:

1. When `dest` is a new file. In your case, only the first iteration `dest` is a new file.
2. When `dest` is overridden. In your case, all the iterations after the first one will override the existing file content.

Reply:
True. I assume it will have a similar output. If this is not a blocker for you, I recommend merging as it is.